src/website/blog/2025-12-04-newsletter.md

```diff
@@ -0,0 +1,141 @@
+great deal of coverage and interest from the community in the new features and
+capabilities. As a reminder, v3.2 provides a host of changes such as the
+refactored and improved [Tag Object](), support for the `QUERY` HTTP Method,
+support for sequential and streaming data protocols, and additional Security
+Scheme features like OAuth 2.0 Device Authorization Flow. You can find out more
+in our
+[blog post](https://www.openapis.org/blog/2025/09/23/announcing-openapi-v3-2)),
+which includes links to key resource to help with upgrading to v3.2. Our
+[Ecosystem Spotlight](#ecosystem-spotlight) also provides resources from the
+community on what the v3.2 upgrade means.
+
+Completing v3.2 means we are now looking at what could appear in our next
+version and beyond, and the team is ramping up for further releases in 2026,
+with an expected v3.3, covering improvements to Security Schemes and greater
+integration with MCP and AI protocols more generally.
+
+[Henry Andrews](https://github.com/handrews) has already pulled together a
+comprehensive plan of what could be in the next version. If you want to look at
+what's in the frame, contribute ideas, or just keep in touch head over to the
+[GitHub Discussion](https://github.com/handrews) on the topic.
+
+> Placeholder for Arazzo updates.
+
+Overlay is also gearing up for a v1.1.0 release, with several features in the
+frame such as clarifications to format interoperability and a new Parameter
+Object in the frame. If you want to contribute or simply find out more about
+what's going on, head over to the
+[Overlay Specification repository](https://github.com/OAI/Overlay-Specification).
+
+### Ecosystem Spotlight
+
+We've already mentioned the release of v3.2 and we've had a great feedback from
+the community. Here's a few samples of what folks are saying.
+
+- [Dave Shanley](https://www.linkedin.com/in/daveshanley/) brought together a
+  really comprehensive [overview](https://quobix.com/articles/openapi-3.2/) of
+  all the new features of v3.2 ("_I love that new feature smell _") with plenty
+  of snippets showing exactly how to implement them.
+
+- [Anton Okolelov](https://www.linkedin.com/in/anton-okolelov/) also
+  [dug deep](https://medium.com/@okoanton/openapi-3-2-0-evolving-with-modern-api-patterns-9977972e9381)
+  in his overview, describing streaming support as "_a long overdue addition_".
+  He summarizes the value of v3.2 in this way: "_While no specification can
+  anticipate every future pattern, OpenAPI 3.2.0 demonstrates a willingness to
+  adapt to observed practices rather than dictating them. That’s a healthy
+  approach for any evolving standard._"
+
+- [Zaid Daba'een](https://www.linkedin.com/in/zaiddabaeen/) describes the update
+  as focusing "_...on the real pain points that show up when you scale: messy
+  docs, patchy auth support, and unclear streaming behavior._"
+
+For the contributors involved in the creation of v3.2, this level of validation
```

Comment:
Maybe note that 3.2 had the most major contributors yet, as seen through new additions to the expanded Editor's list.  idk if @lornajane and @karenetheridge would want this called out or not, and it should be up to them, but for the first time we have two women in the editors list.

Reply:
I'd prefer we not mention the gender aspect